spec/models/work_search_form_spec.rb

```diff
@@ -243,4 +232,29 @@
       end
     end
   end
+
+  describe "sorting results" do
+    describe "by authors" do
+      before do
+        %w(-zzz _yasuho 21st_wombat 007aardvark 6tasmanian_devil).each do |pseud_name|
```

Comment:
My only thought here is that since it's the work method that was broken, maybe we should test the behavior of it primarily in a work spec unit test. Then when we know it's saving the values correctly, we might not need to create as many works here and we can save some runtime that way.